### ISSUE TITLE
Quantization Time ?

### ISSUE THREAD

**Issue by chunseoklee** [facebookresearch/ParetoQ#12]
I just wonder I reproduce the quantization process in local settings.

quote from paper : 

> We employed the AdamW (Loshchilov & Hutter, 2017) optimizer with zero weight decay for optimization. The training was distributed across 16 GPUs, with each GPU handling a batch size of 8. For binary, ternary, and 2-bit quantization settings, the optimization process spanned 120,000 iterations with initial learning rate of 2 × 10−5. For 3-bit and 4-bit settings, the process involved 40,000 iterations with initial learning rate of 1 × 10−5. The learning rate decayed to zero following cosine learning rate decay.

How long it will take during this process in env of papers ?
